lib/event.c

```diff
@@ -879,15 +879,18 @@ static int fd_poll(struct event_loop *m, const struct timeval *timer_wait,
 	}
 
 #if defined(HAVE_PPOLL)
+        /* ppoll supports nanosecond timeout through timespec
```

Comment:
something is fishy with the indentation, I approved the change, but since it looks like CI has to rerun maybe fix this whitespace issue in the meantime.